packages/serverpod/lib/src/database/concepts/transaction.dart

```diff
@@ -8,3 +8,34 @@ abstract interface class Transaction {
   /// exception depending on driver.
   Future<void> cancel();
 }
+
+/// Isolation levels for transactions.
+enum IsolationLevel {
+  /// A statement can only see rows committed before it began.
+  readCommitted,
+
+  /// Transaction may see uncommitted changes made by other transactions.
+  /// Not supported in PostgreSQL and is treated as readCommitted.
+  readUncommitted,
```

Comment:
Should we remove this if it is not supported anyway?

Reply:
Yes maybe, My thinking was that the "interface" should be agnostic to if this is supported in the database or not, while the adapter takes care of it. But I'm not sure there.

Reply:
Or reading a bit more, it is actually possible to set the transaction level `readUncommitted` in postgres, its only that the behavior will be exactly the same as `readCommitted`. So my description here is slightly incorrect.